src/predicates/intersects.jl

```diff
@@ -168,8 +247,18 @@ intersects(m::Multi, c::Chain) = intersects(c, m)
 function minkowskipoint(g₁::Geometry{Dim,T}, g₂::Geometry{Dim,T}, d) where {Dim,T}
   n = Vec{Dim,T}(d[1:Dim])
   v = supportfun(g₁, n) - supportfun(g₂, -n)
-  Point(ntuple(i -> i ≤ Dim ? v[i] : zero(T), max(Dim, 3)))
+  Point(ntuple(i -> i ≤ Dim ? v[i] : zero(T), Dim))
```

Comment:
I believe that we don't need the ternary operator here anymore given that we are now using `Dim` consistently with the dimension of the coordinates? Perhaps we just need to return `v` now?